docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -68,6 +68,11 @@
                         <input type="text" class="form-control" [(ngModel)]="placeholder">
                     </div>
                 </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="resetFilter">Reset filter every 10 seconds</ux-checkbox>
```

Comment:
The customize section is mainly meant for binding actual input properties. This seems more like a test case than something informative to consumers.

I would suggest either leaving the example as it was, or a text input binding directly to the new `filter` input.

Reply:
Unfortunately, the drop-down closes as soon as the user clicks outside. Therefore, a filter text input does not help to see the feature. I have reverted the example.